From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: math: rational: add missing MODULE_DESCRIPTION() macro
Date: Tue, 2 Jul 2024 15:47:59 -0700

With ARCH=sh, make allmodconfig && make W=1 C=1 reports: WARNING: modpost:
missing MODULE_DESCRIPTION() in lib/math/rational.o

Add the missing invocation of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240702-md-sh-lib-math-v1-1-93f4ac4fa8fd@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/math/rational.c |    1 +
 1 file changed, 1 insertion(+)

--- a/lib/math/rational.c~math-rational-add-missing-module_description-macro
+++ a/lib/math/rational.c
@@ -108,4 +108,5 @@ void rational_best_approximation(
 
 EXPORT_SYMBOL(rational_best_approximation);
 
+MODULE_DESCRIPTION("Rational fraction support library");
 MODULE_LICENSE("GPL v2");
_
